Introduction to Federated Learning with Substra.ipynb

```diff
@@ -0,0 +1,1143 @@
+{
```

Comment:
<p>Woups I didn't saw this cell before writting my comment on strategies.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Substra/substra-workshops/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>